src/prefect/client/client.py

```diff
@@ -567,25 +590,21 @@ def get_available_tenants(self) -> List[Dict]:
         """
         Returns a list of available tenants.
 
-        NOTE: this should only be called by users who have provided a USER-scoped API token.
-
         Returns:
             - List[Dict]: a list of dictionaries containing the id, slug, and name of
             available tenants
         """
         result = self.graphql(
             {"query": {"tenant(order_by: {slug: asc})": {"id", "slug", "name"}}},
             # use the API token to see all available tenants
-            token=self._api_token,
+            token=self._api_key,
         )  # type: ignore
         return result.data.tenant  # type: ignore
 
     def login_to_tenant(self, tenant_slug: str = None, tenant_id: str = None) -> bool:
         """
         Log in to a specific tenant
 
-        NOTE: this should only be called by users who have provided a USER-scoped API token.
-
```

Comment:
May be worth noting that this cannot be called if the API Key is from a Service Account

Reply:
it should work with an SA API Key, it will just only return the SA's tenant